.node-version

```diff
@@ -1 +1 @@
-8.15.0
+10.15.1
```

Comment:
what does this file do?

Reply:
I think it was added to specify which node version to use for installing/building/running, but I have no idea if it's enforced in any way.

IMO using the "engines" property in the root `package.json` might be better because that's enforced when you try to run `yarn`, etc.

Reply:
Reasons for the addition of this file are in the original PR https://github.com/codercom/code-server/pull/272

Reply:
Seems reasonable then I guess.